regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -238,7 +241,24 @@ var MainView = Backbone.View.extend({
 
         // change focus to main content area when new sections are loaded
         $('.section-focus').focus();
-        $('table').stickyTableHeaders();
+        this.setHandlers();
+    },
+
+    initHandlers: function() {
+        $.extend($.fn.dataTable.defaults, {
+                paging: false,
+                searching: false,
+                scrollY: 400,
+                scrollX: true,
+                info: false
+        });
+    },
```

Comment:
Yes - it can be added to the actual call if that is preferred.

$('table', this.$el).each(function(index, element) {
         $(element).DataTable({
                paging: false,
                searching: false,
                scrollY: 400,
                scrollX: true,
                info: false
        });       
});